src/services/helpers/service.js

```diff
@@ -29,12 +30,12 @@ module.exports = function setup(app) {
 
 		// POST
 		async create(data, params) {
-			const FORCE_SEND_EMAIL = app.get('FORCE_SEND_EMAIL');
-			const notificationPlatform = app.get('NOTIFICATION_PLATFORM');
+			const FORCE_SEND_EMAIL = Configuration.get('FORCE_SEND_EMAIL');
+			const notificationPlatform = Configuration.get('NOTIFICATION_PLATFORM');
 
-			if (!notificationPlatform) {
+			/* if (!notificationPlatform) {
 				throw new MethodNotAllowed('Required Env NOTIFICATION_PLATFORM is not defined');
-			}
+			} */
```

Comment:
Can be removed. The Configuration will throw in line 34 if NOTIFICATION_PLATFORM is not defined

Reply:
But this throw is blocked the FORCE_SEND_EMAIL logic.